README.md

```diff
@@ -34,32 +34,7 @@ For a full walkthrough of building custom images, use this [guide.](./doc/guides
 A brief video demonstration of building and using the device service can be found [here.](https://www.youtube.com/watch?v=vZqd3j2Zn2Y)
 
 # Learn More 
-
```

Comment:
Shouldn't the above sections bee remove also and just reference the Docs. Guide links above are broken.